pkg/auth/handler/webapp/authflowv2/change_password.go

```diff
@@ -99,6 +99,7 @@ func (h *AuthflowV2ChangePasswordHandler) ServeHTTP(w http.ResponseWriter, r *ht
 			return err
 		}
 
+		h.Controller.Navigator.NavigateChangePasswordSuccessPage(screen, r, s.ID, result)
```

Comment:
Make navigator a direct dependency of this handler, instead of using it through the controller.

Reply:
Any reason we shouldn't do that?

Reply:
Some reasons we don't want to do it:
1. If we use `Controller.Navigator`, we assumes `Controller` is a struct with a property `Navigator`. We might not want to make such assumption (Maybe we want to change it into an interface in the future)
2. In my opinion, If you call `Navigator.NavigateChangePasswordSuccessPage` in the handler, it means the thing that "needs" a `Navigator` is the handler. So it should be a direct dependency of the handler.